ide/coqide/coqide.ml

```diff
@@ -1614,13 +1621,13 @@ let build_ui () =
 
   menu help_menu [
     item "Help" ~label:"_Help";
-    item "Browse Coq Manual" ~label:"Browse Coq _Manual"
+    item "Browse Coq Manual" ~label:"Browse Coq _Manual" ~accel:"<Shift>F1" ~stock:`HELP
```

Comment:
I don't see any icons on the Help menu on my system (Windows/WSL with Ubuntu/XWindows).